samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -4,29 +4,32 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-&mram1x {
-	/delete-node/ cpuapp_rw_partitions;
+/delete-node/ &cpuapp_rw_partitions;
 
+&mram1x {
 	cpuapp_rw_partitions: cpuapp-rw-partitions {
 		compatible = "nordic,owned-partitions", "fixed-partitions";
 		status = "okay";
-		perm-read;
-		perm-write;
-		perm-secure;
+		nordic,access = <NRF_OWNER_ID_APPLICATION NRF_PERM_RWS>;
 		#address-cells = <1>;
 		#size-cells = <1>;
 
 		dfu_partition: partition@100000 {
 			reg = <0x100000 DT_SIZE_K(908)>;
 		};
 
-		storage_partition: partition@1e3000 {
-			reg = <0x1e3000 DT_SIZE_K(20)>;
+		/* Align the partition size to 4096 B to be able to enable the
```

Comment:
`... to avoid gaps.`? I am not sure if nRF54H will actually support fprotect (currently fprotect seems not supported) so maybe we could desist from mentioning it explicitly.